stemcell_builder/stages/system_google_packages/apply.sh

```diff
@@ -7,21 +7,19 @@ set -e
 base_dir=$(readlink -nf $(dirname $0)/../..)
 source $base_dir/lib/prelude_apply.bash
 
-# Copy google daemon packages into chroot
-cp -R $assets_dir/usr $chroot/
-
-# Configure the Google guest environment
-# https://github.com/GoogleCloudPlatform/compute-image-packages#configuration
-cp $assets_dir/instance_configs.cfg.template $chroot/etc/default/
-
 os_type="$(get_os_type)"
 if [ "${os_type}" == "ubuntu" ]
 then
-  # Run google-accounts-manager and google-clock-sync-manager with upstart
-  cp $assets_dir/etc/init/google-accounts-manager-{task,service}.conf $chroot/etc/init/
-  cp $assets_dir/google-address-manager.conf $chroot/etc/init/
-  cp $assets_dir/google-clock-sync-manager.conf $chroot/etc/init/
-  chmod 0644 $chroot/etc/init/google*
+  # Copy google daemon packages into chroot
+  cp -R $assets_dir/*.deb $chroot/tmp/
+
+  # Configure the Google guest environment
+  # https://github.com/GoogleCloudPlatform/compute-image-packages#configuration
+  cp $assets_dir/instance_configs.cfg.template $chroot/etc/default/
+
+  run_in_chroot $chroot "apt-get update"
+  run_in_chroot $chroot "apt-get install -y python-setuptools python-boto"
+  run_in_chroot $chroot "dpkg --force-all -i /tmp/*.deb"
```

Comment:
Does it make sense to put these debs into a folder /tmp/google_debs dir so that we're not installing a deb in the tmp directory we did not intend to install?